style_guide.md

```diff
@@ -108,6 +106,16 @@ Don't tolerate warnings and don't suppress them due to laziness.
 
 Use the following source file layouts.
 
+Precede comments with a blank line.
```

Comment:
Right, this has been applied fairly consistently, though possibly not in some sections that haven't been hit in a long time. In order to open up the code vertically, and to do so consistently, I applied two commonly applied rules: scopes and comments are preceded by blank lines.